### PR TITLE
add PR number to and touch up NEWS.md entry for #18754

### PR DESCRIPTION
**Pull request by Sacha0**
Ref. #21475. Best!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -397,13 +397,13 @@ Compiler/Runtime improvements
...
-    removing the need for special code-generator support for Intrinsics.
+    removing the need for special code-generator support for `Intrinsics` ([#18754]).
 
   * `ccall` gained limited support for a `llvmcall` calling-convention.
-    This can replace many uses of `llvmcall` with a simpler, shorter declaration.
+    This can replace many uses of `llvmcall` with a simpler, shorter declaration ([#18754]).
 
-  * All Intrinsics are now Builtin functions instead and have proper error checking
-    and fall-back static compilation support.
+  * All `Intrinsics` are now `Builtin` functions instead and have proper error checking
```
Comment (tkelman) [inline NEWS.md]: I don't think these have code meanings, should maybe just be not capitalized

File: NEWS.md [hunk 2]
```diff
@@ -584,3 +585,4 @@ Deprecated or removed
 [#20543]: https://github.com/JuliaLang/julia/issues/20543
 [#20609]: https://github.com/JuliaLang/julia/issues/20609
 [#20889]: https://github.com/JuliaLang/julia/issues/20889
+[#21359]: https://github.com/JuliaLang/julia/issues/21359
```
Comment (tkelman) [inline NEWS.md]: this part of the diff shouldn't be backported

File: NEWS.md (hunk 1 above)
Comment (Sacha0) [inline NEWS.md]: Removed quoting and capitalization. Thanks!

File: NEWS.md (hunk 2 above)
Comment (Sacha0) [inline NEWS.md]: Good catch! I stripped this line from the diff here and elsewhere. I will strip this line from any future commits as well. Thanks!

File: NEWS.md (hunk 1 above)
Comment (tkelman) [inline NEWS.md]: oh they exist in Core, but are very sparsely documented in the devdocs

Comment (Sacha0) [inline NEWS.md]: Quoting and capitalization restored. How now? :)

Comment (tkelman) [inline NEWS.md]: pretty sure the module name is plural

Comment (Sacha0) [inline NEWS.md]: Fixed. Thanks!

**Pull request merged**